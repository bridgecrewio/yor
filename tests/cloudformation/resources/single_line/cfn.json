{
  "AWSTemplateFormatVersion": "2010-09-09",

  "Parameters": {
    "BucketName": {
      "Type": "String",
      "Default": "bucket-name"
    }
  },

  "Resources": {
    "CLOUDFORMATIONNP": {
      "Type": "AWS::S3::Bucket",
      "DeletionPolicy": "Delete",
      "Properties": {
        "BucketName": { "Ref": "BucketName" },
        "BucketEncryption": {
          "ServerSideEncryptionConfiguration": [
            {
              "ServerSideEncryptionByDefault": {
                "SSEAlgorithm": "AES256"
              }
            }
          ]
        },
        "Tags": [
          { "Key": "old_tag1", "Value": "old_val1" },
          { "Key": "old_tag2", "Value": "old_val2" },
          {
           "Key": "yor_trace",
           "Value": "673453f2-e3c4-4056-832c-1fdcdba928a4"
          }
        ]
      }
    }
  }
}
