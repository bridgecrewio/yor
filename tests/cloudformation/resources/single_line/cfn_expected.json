{
  "AWSTemplateFormatVersion": "2010-09-09",

  "Parameters": {
    "BucketName": {
      "Type": "String",
      "Default": "bucket-name"
    }
  },

  "Resources": {
    "CLOUDFORMATIONNP": {
      "Type": "AWS::S3::Bucket",
      "DeletionPolicy": "Delete",
      "Properties": {
        "BucketName": { "Ref": "BucketName" },
        "BucketEncryption": {
          "ServerSideEncryptionConfiguration": [
            {
              "ServerSideEncryptionByDefault": {
                "SSEAlgorithm": "AES256"
              }
            }
          ]
        },
        "Tags": [
          { "Key": "old_tag1", "Value": "old_val1" },
          { "Key": "old_tag2", "Value": "old_val2" },
          {
           "Key": "new_tag",
           "Value": "new_value"
          },
          {
           "Key": "another_tag",
           "Value": "another_val"
          }
        ]
      }
    }
  }
}
